### PR TITLE
Update Alerting Policies (Node Level)

### PR DESCRIPTION
**Pull request by Felixnoo**
Signed-off-by: Felixnoo <felixliu@yunify.com>

### CODE REVIEW

**Comment by Felixnoo** [comment]
fix #1438

**Review by benjaminhuo — APPROVED** [review]
/lgtm

File: content/en/docs/cluster-administration/cluster-wide-alerting-and-notification/alerting-policy.md [hunk 1]
```diff
@@ -6,94 +6,62 @@ linkTitle: "Alerting Policies (Node Level)"
 weight: 8530
 ---
 
-## Objective
-
-KubeSphere provides alerting policies for nodes and workloads. This guide demonstrates how you can create alerting policies for nodes in the cluster and configure mail notifications. See [Alerting Policy (Workload Level)](../../../project-user-guide/alerting/alerting-policy/) to learn how to configure alerting policies for workloads.
+KubeSphere provides alerting policies for nodes and workloads. This tutorial demonstrates how to create alerting policies for nodes in a cluster. See [Alerting Policy (Workload Level)](../../../project-user-guide/alerting/alerting-policy/) to learn how to configure alerting policies for workloads.
```
Comment (Sherlock113) [inline content/en/docs/cluster-administration/cluster-wide-alerting-and-notification/alerting-policy.md:9]: Add another paragraph here:

KubeSphere also has built-in policies which will trigger alerts if conditions defined for these policies are met. On the **Built-in Policies** tab, you can click a policy to see its details. Note that they cannot be directly deleted or edited on the console.

**Pull request merged**